lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -464,6 +469,7 @@ def destroy(self, *args):
     def show(self):
         # show the figure window
         self.window.show()
+        self.window.present()
```

Comment:
The only real change in functionality that I can see I have made, but logically this feels right to me, that ``show`` should always bring the window to the front, whether or not it had already been opened or not.  Though feel free to disagree.